### PR TITLE
Updated RC doc for enabling RC on agent by default

### PR DESCRIPTION
**Pull request by bawsome**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/remote_config/_index.md [hunk 1]
```diff
@@ -44,12 +44,12 @@ The following diagram illustrates how Remote Configuration works:
 
 1. You configure select product features in the Datadog UI.
 2. The product feature configurations are securely stored within Datadog.
-3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog.
+3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog. Tracing libraries, deployed in your environments, communicate with Agents to request and receive configuration updates from Datadog.
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:47]:
```suggestion
3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog.
4. Tracing libraries, deployed in your environments, communicate with Agents to request and receive configuration updates from Datadog.
```

File: content/en/agent/remote_config/_index.md [hunk 2]
```diff
@@ -44,12 +44,12 @@ The following diagram illustrates how Remote Configuration works:
...
 
 **Note**: Configuration changes applied through Remote Configuration are not shown in your Agent configuration file.
 
-## Product and feature capabilities
-The following products and features are supported with Remote Config:
+## Support products and feature capabilities
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:51]:
```suggestion
## Supported products and feature capabilities
```

File: content/en/agent/remote_config/_index.md [hunk 3]
```diff
@@ -98,15 +98,8 @@ Datadog implements the following safeguards to protect the confidentiality, inte
...
-
 - Datadog Agent version `7.41.1`  (`7.42.0` for APM sampling rate, `7.43.0` for APM Remote Instrumentation) or higher installed on your hosts or containers. 
-- For features that use tracing libraries, the following minimum versions of Datadog tracing libraries include:
-
-  | Product feature                        | Go            | Java          | .Net          | NodeJS
-  |----------------------------------------|---------------|---------------|---------------|---------------|
-  | Dynamic Instrumentation |               | 1.5.0         | 2.22.0        |               |
-
-  For ASM Protection capabilities and ASM 1-click activation, see [Compatibility Requirements][6].
+- For Datadog products that use tracing libraries, you also need to upgrade your tracing libraries to a Remote Configuration compatible version. For ASM Protection capabilities and ASM 1-click activation, see [Compatibility Requirements][6]. For Dynamic Instrumentation, see [Compatibility Requirements][20].
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:102]:
```suggestion
- For Datadog products that use tracing libraries, you also need to upgrade your tracing libraries to a Remote Configuration-compatible version. For ASM Protection capabilities and ASM 1-click activation, see [ASM compatibility requirements][6]. For Dynamic Instrumentation, see [Dynamic Instrumentation prerequisites][20].
```

File: content/en/agent/remote_config/_index.md [hunk 4]
```diff
@@ -118,15 +111,16 @@ To enable Remote Configuration:
...
 3. On the [Remote Configuration][8] page, enable Remote Configuration. This enables Datadog components across your organization to receive configurations from Datadog.
 
-4. Select an existing API key or create a new API key, and enable the Remote Config capability on the key:
+4. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key:
 
-   {{<img src="agent/remote_config/RC_Key_updated.png" alt="API Key properties with Remote Config capability Enable button." width="90%" style="center">}}
+   {{<img src="agent/remote_config/RC_Key_updated.png" alt="API Key properties with Remote Configuration capability Enable button." width="90%" style="center">}}
 
 5. Update your Agent configuration file:
+Note: This step is required only for Agent versions 7.46.0 or lower. Starting with Agent version 7.47.0, remote_configuration.enabled is set to true by default in the Agent.
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:119]:
```suggestion
**Note:** This step is required only for Agent versions 7.46.0 or lower. Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent.
```

File: content/en/agent/remote_config/_index.md [hunk 5]
```diff
@@ -204,10 +198,43 @@ Gain visibility into the Remote Configuration status of your Agent through the [
...
   | NOT CONNECTED       | The Agent cannot be found in the Remote Configuration service and could have `remote_config.enabled` set to true or false in its `datadog.yaml` configuration file. Check your local Agent configuration or your proxy settings. This status displays when the Agent version is higher than `7.41.1` but lower than `7.45.0`.            | 
   | UNSUPPORTED AGENT   | The Agent is on a version that is not Remote Configuration capable. To fix this issue, update the Agent to the latest available version. |
+
+## Opting out of Remote Configuration at the Agent level
+
+Starting with Agent version 7.47.0, remote_configuration.enabled is set to true by default in the Agent. This will cause the agent to request configuration updates from the Datadog site. To receive configurations from Datadog, you still need to enable Remote Configuration at the organization level and enable Remote Configuration capability on your API Key from the Datadog UI, and allow outbound HTTPS access to Remote Configuration [endpoints][17] from your environment. If you don't want your agent to send configuration requests to Datadog, you can set remote_configuration.enabled to false in the Agent.
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:208]:
```suggestion
Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent. This setting causes the Agent to request configuration updates from the Datadog site.

To receive configurations from Datadog, you need to take the following steps:
- Enable Remote Configuration at the organization level.
- Enable Remote Configuration capability on your API Key from the Datadog UI.
- Allow outbound HTTPS access to Remote Configuration [endpoints][17] from your environment.

If you don't want your Agent to send configuration requests to Datadog, you can set `remote_configuration.enabled` to `false` in the Agent.
```

File: content/en/agent/remote_config/_index.md [hunk 6]
```diff
@@ -204,10 +198,43 @@ Gain visibility into the Remote Configuration status of your Agent through the [
...
+
+{{< tabs >}}
+{{% tab "Configuration YAML file" %}}
+Change remote_configuration.enabled from true to false in your configuration YAML file:
```
Comment (urseberry) [inline content/en/agent/remote_config/_index.md:212]:
```suggestion
Change `remote_configuration.enabled` from `true` to `false` in your configuration YAML file:
```

Comment (urseberry) [inline content/en/agent/remote_config/_index.md:212]: Can you add any more information about the configuration file? What the filename is, and where it is located, perhaps?

File: content/en/agent/remote_config/_index.md (hunk 5 above)
Comment (bawsome) [reply content/en/agent/remote_config/_index.md:209]: Can we add "also" to the sentence? Reason being we want to state that in addition to setting RC to true in the agent, customers also need to take additional steps to fully enable RC. 

To receive configurations from Datadog, you also need to take the following steps:

File: content/en/agent/remote_config/_index.md (hunk 6 above)
Comment (bawsome) [reply content/en/agent/remote_config/_index.md:213]: The filename is datadog.yaml and location varies. We have a link in our page here:
https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6v7#agent-main-configuration-file
I will link to it.

**Review by bawsome — COMMENTED** [review]
Submitting again with the said changes. Note for the steps below the diagram, we'd like to have only 3 steps instead of 4 as the diagram only shows 3 steps.

**Review by urseberry — APPROVED** [review]

**Pull request merged**